airbyte-workers/src/main/java/io/airbyte/workers/DefaultNormalizationWorker.java

```diff
@@ -39,7 +40,7 @@ public DefaultNormalizationWorker(final String jobId,
   }
 
   @Override
-  public Void run(final NormalizationInput input, final Path jobRoot) throws WorkerException {
+  public StandardNormalizationSummary run(final NormalizationInput input, final Path jobRoot) throws WorkerException {
```

Comment:
After addressing the other comments in this PR, I think it would be a good idea to deploy this to a dev environment and run a sync that performs normalization, then check the `attempts` table for that dev env to verify that the normalizationSummary is present in the corresponding attempt record